devtools-ci-index.yaml

```diff
@@ -1586,6 +1586,69 @@
         - registry_devshift_credentials
     <<: *job_template_build_defaults
 
+- job-template:
+    name: '{ci_project}-planner-api-test'
+    triggers:
+      - timed: '0 0 * * * *'
+    wrappers:
+      - credentials-binding:
+        - text:
+            credential-id: 1b1d24cc-7bf5-46dc-a112-657de8f1388c
+            EE_TEST_OSO_TOKEN: Token
+      - ansicolor
+    concurrent: false
+    scm:
```

Comment:
Do we need this here? I think it should be brought in by line 1647

Reply:
done